packages/form/src/TextField.tsx

```diff
@@ -51,13 +53,39 @@ const TextField = forwardRef<HTMLInputElement, TextFieldProps>(
       );
     }
 
+    if (autoComplete === 'off') {
+      return (
+        <input
+          {...rest}
+          data-lpignore="true" // "data-lpignore" is added to prevent LastPass from injecting a password autofill icon
+          data-1p-ignore // "data-1p-ignore" is added to prevent 1Password from injecting a password autofill icon
```

Comment:
Could we instead conditionally apply these attributes based off `autoComplete` to the template below to minimize duplication?

Reply:
Yeah! I'll update :)